ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -159,8 +158,8 @@ const EnableCtSwitch = (props: {
       data-test-id='controlled-enabled-switch'
       onChange={(v) => onChange(v.value)}
       checked={
-        getControlledTierConfig(institution).membershipRequirement !==
```

Comment:
removed tier-specific versions of this, since this was the only remaining reference